Trabalho 2/entities/factory2/line/main.py

```
@@ -0,0 +1,138 @@
+import time
+import os
+import sys
+from shared.mqtt_client import get_client
+from shared.products import BOM
+import threading
+
+# --- Configurações da Linha ---
+LINE_ID = os.getenv("LINE_ID")
+
+if not LINE_ID:
+    print(f"[LINE] ERRO: Variável de ambiente LINE_ID não definida.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)